Makefile

```diff
@@ -133,7 +133,14 @@ endef
 # run test examples
 define test_examples
     @echo "# Testing examples"
-    @./scripts/test_examples.sh
+    @./scripts/test_examples/test_examples.sh
+    @echo "# done"
+endef
+
+# run test examples2
+define test_examples2
+    @echo "# Testing examples: reactions to disconnect/reconnect of plugins redis, cassandra ..."
```

Comment:
- call both of the scripts from test_examples (remove test_examples2)
- rename reactions_to_reconnecting_of_plugins to plugin_reconnect (or something shorter)
- then in test-examples (line 165) call just test_examples